### PR TITLE
remove old workflow state fields

### PR DESCRIPTION
**Pull request by tpolecat**
This removes `ObsStatus`, `ObsActiveStatus`, and the `forReview` field, all of which are subsumed by `ObservationWorkflowState`. Associated tests for these properties'

A followup PR will add an internal user state that allows the user to set the workflow state to `Inactive` and `Ready` when appropriate.

Explore should remove references to these things in preparation for the schema change.

### CODE REVIEW

File: modules/service/src/test/scala/lucuma/odb/graphql/query/observationValidations.scala [hunk 1]
```diff
@@ -1,690 +0,0 @@
-// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```
Comment (tpolecat) [inline modules/service/src/test/scala/lucuma/odb/graphql/query/observationValidations.scala:1]: The `validations` field was removed and is now on `workflow.validationErrors`. The associated tests (in `observation_workflow`) include the tests here.

File: modules/service/src/test/scala/lucuma/odb/graphql/mutation/cloneObservation.scala [hunk 2]
```diff
@@ -228,96 +222,96 @@ class cloneObservation extends OdbSuite {
 
   }
 
-  test("cloned observation should always be present/new/active") {
+  // test("cloned observation should always be present/new/active") {
```
Comment (toddburnside) [inline modules/service/src/test/scala/lucuma/odb/graphql/mutation/cloneObservation.scala:225]: Do we want to keep this code? If so, maybe comment on why it is commented out. Same for the commented out tests in the next few files.

**Review by toddburnside — APPROVED** [review]
LGTM. I'd almost approve it on the basis of this alone: 
image
Deleted lines could be even higher if you delete commented out tests. :)

File: modules/service/src/test/scala/lucuma/odb/graphql/mutation/cloneObservation.scala (hunk 2 above)
Comment (tpolecat) [reply modules/service/src/test/scala/lucuma/odb/graphql/mutation/cloneObservation.scala:225]: Yeah I will go back over these this morning and see if they're still relevant.

**Pull request merged**